pom.xml

```diff
@@ -377,10 +376,6 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-failsafe-plugin</artifactId>
           <version>2.22.2</version>
-          <configuration>
-            <skipITs>${skipITs}</skipITs>
-            <argLine>${failsafe.argLine}</argLine>
-          </configuration>
```

Comment:
Why did we get rid of this? Is it not needed?

Reply:
The ITs were running even if '-DskipTests' was specified, which was a negatively unexpected outcome. Normally, the failsafe plugin will honor that first and then honor '-DskipITs' second, but the way the plugin was configured broke that connection so my ITs were _always_ running despite using common build idioms.

Also, the only thing dictating which environment to hit is the environment variable as opposed to anything related to the failsafe plugin, and the massive failsafe configured didn't seem to be adding any value from my perspective, so I thought that reducing that and just using env values in our builds was a simpler and cleaner twiddle than having a bunch of failsafe configs.

Any objections?

It's occurring to me that we might need to update the GH actions here though.

Reply:
Nope no objections. Just wanted to make this this wasn't an inadvertent deletion.

I think as long as ITs are running in CI, we're probably fine. I don't think we are skipping ITs in CI so this shouldn't affect that